### PR TITLE
tags/usecases/cateogires 366-410

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
relates to: https://jira-hq.paloaltonetworks.local/browse/CIAC-4318

### CODE REVIEW

File: Packs/HackerOne/pack_metadata.json [hunk 1]
```diff
@@ -10,14 +10,9 @@
     "categories": [
         "Vulnerability Management"
     ],
-    "tags": [
```
Comment (martijnrusschen) [inline Packs/HackerOne/pack_metadata.json:13]: Why are these removed?

File: Packs/HackerOne/pack_metadata.json [hunk 2]
```diff
@@ -10,14 +10,9 @@
...
-        "Security"
-    ],
-    "useCases": [
```
Comment (martijnrusschen) [inline Packs/HackerOne/pack_metadata.json:16]: Same here

File: Packs/HackerOne/pack_metadata.json [hunk 3]
```diff
@@ -10,14 +10,9 @@
...
-        "Vulnerability Management"
-    ],
+    "tags": [],
+    "useCases": [],
     "keywords": [
-        "HackerOne",
```
Comment (martijnrusschen) [inline Packs/HackerOne/pack_metadata.json:20]: Why is this removed?

File: Packs/HackerOne/pack_metadata.json (hunk 1 above)
Comment (bakatzir) [reply Packs/HackerOne/pack_metadata.json:13]: We are revamping the entire tags and use cases sections for the XSOAR and XSIAM Marketplaces, as they have had too many options that have cluttered the search options.

File: Packs/HackerOne/pack_metadata.json (hunk 3 above)
Comment (bakatzir) [reply Packs/HackerOne/pack_metadata.json:20]: keywords are added to the pack name words, so it is a redundancy.

Comment (bakatzir) [reply Packs/HackerOne/pack_metadata.json:20]: BTW, the pack description is also used in the search bar.

**Review by GuyLibermanPA — APPROVED** [review]
